### DISCUSSION TITLE
Idasen compatibility?

### DISCUSSION THREAD

**Discussion by chaineym** [gcormier/megadesk#95] [Q&A] [answered]
Hey guys, that project looks very interesting. I was wondering if it would work with IKEA's Idasen desks. The controllers look different but maybe the internals are the same.

**Comment by gcormier** [comment, marked as answer]
Check out #21

**Reply by chaineym** [reply]
Ah I see, sorry I should've checked. I still think it would be nicer to have it in Home Assistant VS relying on a 3rd party app but I understand it's a valid option. Thanks for the info!

**Reply by gcormier** [reply]
https://github.com/mitsuhiko/idasen-control
